charts/portworx/values.yaml

```diff
@@ -21,7 +25,7 @@ managementInterface: none             # Name of the interface <ethX>
 envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
 
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 2.7.0
+storkVersion:                         # Version of Stork. For eg: 2.7.0
```

Comment:
We need a default value

Reply:
Operator uses manifest to fetch the version, we don't need to provide the value by default.

Reply:
If we don't this this, can we remove the field? Leaving it blank may confuse users since `values.yaml` is used to provide the default value.

Reply:
the behavior is different, when stork version is not provided, operator will get stork version according to px version based on version server or configmap (in airgapped environment). if stork version is provided, then it's pinned to the version, it won't change if px version changes. users can choose to pin it or not with this value.